CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 5.0.5
```

Comment:
oops, I just noticed I failed to update the version correctly in my CL...

Reply:
Fixed!